lib/trino-plugin-toolkit/pom.xml

```diff
@@ -33,6 +33,11 @@
             <artifactId>configuration</artifactId>
```

Comment:
Please make sure that your commit messages follow: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages